src/app/services/upload/upload.service.ts

```diff
@@ -0,0 +1,24 @@
+import { Injectable } from '@angular/core';
+import { RequestService } from '../request/request.service';
+
+@Injectable()
+export class UploadSerice {
+  private requestService: RequestService = new RequestService();
+
+  public uploadToAmazon(response: string, file: File) {
+    const responseJSON: SigningResponse = JSON.parse(response);
+    const formData = new FormData();
+
+    Object.keys(responseJSON.data).forEach(key => {
+      formData.append(key.replace(/_/g, '-'), responseJSON.data[key]);
```

Comment:
I don't know how to resolve:
`Element implicitly has an 'any' type because type '{ key: string; acl: string; policy: string; x_amz_algorithm: string; x_amz_credential: string; x_...' has no index signature.`